### PR TITLE
Make start and end date configurable

### PR DESCRIPTION
**Pull request by masap**
We can use `-s` or `-e` to specify the start and end of logging. However, in many cases, it is the customer who does the logging, and `-s` and `-e` may be inconvenient. Therefore, this PR allows you to specify a start date and time.

Signed-off-by: Masashi Honma <masashi.honma@gmail.com>

### CODE REVIEW

File: collection/rancher/v2.x/logs-collector/rancher2_logs_collector.sh [hunk 1]
```diff
@@ -797,6 +799,14 @@ while getopts "c:d:s:e:r:fph" opt; do
       UNTIL_FLAG="--until ${END_LOGGING}"
       techo "Logging until $END_LOGGING"
       ;;
+    S)
+      SINCE_FLAG="--since ${OPTARG}"
+      techo "Logging since ${OPTARG}"
```
Comment (leodotcloud) [inline collection/rancher/v2.x/logs-collector/rancher2_logs_collector.sh:804]:
```suggestion
      techo "Collecting logs starting ${OPTARG}"
```

File: collection/rancher/v2.x/logs-collector/rancher2_logs_collector.sh [hunk 2]
```diff
@@ -797,6 +799,14 @@ while getopts "c:d:s:e:r:fph" opt; do
...
+      ;;
+    E)
+      UNTIL_FLAG="--until ${OPTARG}"
+      techo "Logging until ${OPTARG}"
```
Comment (leodotcloud) [inline collection/rancher/v2.x/logs-collector/rancher2_logs_collector.sh:808]:
```suggestion
      techo "Collecting logs until ${OPTARG}"
```

**Review by leodotcloud — CHANGES_REQUESTED** [review]

File: collection/rancher/v2.x/logs-collector/rancher2_logs_collector.sh (hunk 1 above)
Comment (masap) [reply collection/rancher/v2.x/logs-collector/rancher2_logs_collector.sh]: Thank you, fixed.

File: collection/rancher/v2.x/logs-collector/rancher2_logs_collector.sh (hunk 2 above)
Comment (masap) [reply collection/rancher/v2.x/logs-collector/rancher2_logs_collector.sh]: Thank you, fixed.

**Review by leodotcloud — APPROVED** [review]

**Pull request merged**